.circleci/config.yml

```diff
@@ -19,38 +19,33 @@ commands:
       - run:
           name: Setup gradle.properties
           command: cp gradle.properties-example gradle.properties
-  yarn-install:
+  npm-install:
     steps:
       - restore_cache:
-          name: Restore Yarn Cache
+          name: Restore NPM Cache
           keys:
-            - yarn-i18n-v4-cache-v{{ .Environment.CACHE_TRIGGER_VERSION }}-job-{{ .Environment.CIRCLE_JOB }}-{{ checksum "libs/gutenberg-mobile/yarn.lock" }}
+            - npm-i18n-v1-cache-v{{ .Environment.CACHE_TRIGGER_VERSION }}-job-{{ .Environment.CIRCLE_JOB }}-{{ checksum "libs/gutenberg-mobile/package-lock.json" }}
       - run:
-          name: Yarn Install
+          name: NPM Install
           working_directory: libs/gutenberg-mobile
-          command: yarn install --frozen-lockfile --prefer-offline --network-concurrency 1
+          command: npm ci --prefer-offline
       - save_cache:
-          name: Save Yarn Cache
-          key: yarn-i18n-v4-cache-v{{ .Environment.CACHE_TRIGGER_VERSION }}-job-{{ .Environment.CIRCLE_JOB }}-{{ checksum "libs/gutenberg-mobile/yarn.lock" }}
+          name: Save NPM Cache
+          key: npm-i18n-v1-cache-v{{ .Environment.CACHE_TRIGGER_VERSION }}-job-{{ .Environment.CIRCLE_JOB }}-{{ checksum "libs/gutenberg-mobile/package-lock.json" }}
           paths:
-            - libs/gutenberg-mobile/node_modules
+            - ~/.npm
```

Comment:
Not a change during this PR but let me just check @ceyhun (or @Tug ), was this removal of the node_modules from caching intentional?

Reply:
It was intentional, based on the suggestion here: https://github.com/actions/cache/blob/master/examples.md#node---npm
> Note: It is not recommended to cache node_modules, as it can break across Node versions and won't work with npm ci
But I didn't have a chance to try if it breaks on CircleCI.